### PR TITLE
Weekly adventure design

### LINKED ISSUE #192: Weekly colors
Edit the color theme depending on the week in the adventure page. The colors are fösets capes in the following order:
- Week 0: #4ab6f4
- Week 1: #da2550
- Week 2: #61192f
- Week 3: #2c370e
- Week 4: #eb7125

### PR DESCRIPTION
**Pull request by FredrikLastow**
- Adds weekly design for the adventures and the navbar on other pages. The colors are also updated from what is stated in the issue to be a bit more UI friendly (Closes #192)
- Adds a hue-rotate egg 
- Changes the weekly progress bars to be inside a slider
- New function `Date.Prototype.getWeekNumber()` that returns the date's current week number 1-52.
- Updated title on adventure page to foremost be the introduction week number and not the actual week number

P.S Hound, you are a bitch in advance

### CODE REVIEW

File: www/scss/partials/_nollning.scss [hunk 1]
```diff
@@ -160,15 +178,15 @@
...
 
 @font-face {
-  font-family: Atlantis;
-  src: url("../fonts/Atlantisnollning.otf");
+  font-family: Planewalker;
+  src: url("../fonts/Planewalker.otf");
+}
+
+#view-nollning .test.navbar {
+  background: none !important;
```
[the remark this answers is not in the document]
Comment (Lullebullelukas) [reply www/scss/partials/_nollning.scss]: Please hound just this once

File: www/js/adventures.js [hunk 2]
```diff
@@ -324,14 +337,30 @@ $$(document).on('page:init', '.page[data-name="adventures"]', function (e) {
             shape.path.setAttribute('stroke-width', state.width);
 
             var value = Math.round(shape.value() * missionsCount);
```
Comment (olofenglund96) [inline www/js/adventures.js]: `const`?

File: www/js/nollning.js [hunk 3]
```diff
@@ -1,34 +1,59 @@
...
+$$(document).on('page:afterin', '.page[data-name="nollning"]', function (e) {
+  const navbar = $('#view-nollning .navbar');
+  if (!navbar.hasClass('nollning-navbar')) {
+    navbar.addClass('nollning-navbar');
+  }
+});
+
 $$(document).on('page:init', '.page[data-name="nollning"]', function (e) {
-  var tab = $(e.detail.view.$el[0]);
+  var tab = $('#view-nollning');
```
Comment (olofenglund96) [inline www/js/nollning.js]: `const`

File: www/js/nollning.js [hunk 4]
```diff
@@ -1,34 +1,59 @@
...
       nollningMoose.addClass('nollning-moose-egg');
     }
   }
 
+  // Append the correct weekly styling (nollning start on week 35)
+  const nollningWeek = new Date().getWeekNumber() - 34;
+  $('#view-nollning').addClass(`adventure-week-${nollningWeek}`);
+
   // Toggle the nollning-toolbar class so the toolbar changes color in the nollnings tab
   var toolbar = $('.toolbar');
```
Comment (olofenglund96) [inline www/js/nollning.js]: `const`

File: www/scss/partials/_nollning.scss [hunk 5]
```diff
@@ -118,7 +118,7 @@
   background-image: url("../img/nollning_moose_icon_active.png");
 }
 
-#view-nollning .navbar {
+#view-nollning:not(.adventure-week-*) .navbar {
```
[the remark this answers is not in the document]
Comment (olofenglund96) [reply www/scss/partials/_nollning.scss]: Sick selector yo

**Pull request merged**